dev/cnf/oss_dependencies.maven

```diff
@@ -624,6 +623,7 @@ org.glassfish.jaxb:jaxb-core:3.0.1
 org.glassfish.jaxb:jaxb-core:4.0.1
 org.glassfish.jaxb:jaxb-jxc:3.0.1
 org.glassfish.jaxb:jaxb-jxc:4.0.1
+org.glassfish.jaxb:jaxb-runtime:2.3.5
```

Comment:
As with the others, this is not the latest 2.x version ... there is a 2.3.9

Reply:
This one is intentional, it's the only non 2.2 version of JAXB that works with the version of CXF that the `wadl2java` tool uses. Not to say it can't be fixed in the future, but part of what took so long to get this work done was finding a semi-acceptable solution for wadl2java. The JAX-RS team asked me to use this version for now, and let them figure out how to upgrade it in the future. I realize that's kicking the can down the road, but it was a can they asked me to kick haha.